### PR TITLE
docs: build page with mkdocs

### PR DESCRIPTION
**Pull request by WanzenBug**
In order to replace the outdated docs on piraeus.io, rebuild the existing documentation for the operator using mkdocs. This should eventually be integrated into CI.

This change makes the docs a bit less readable compared to pure markdown. While not ideal, it makes the docs much more pleasant to read once build.

We can simply link users directly to the new docs. Especially once this is integrated into CI.

### CODE REVIEW

**Comment by WanzenBug** [comment]
This powers our new docs page: https://piraeus.io/docs

For the most part this is just using the default settings for https://www.mkdocs.org/ and https://squidfunk.github.io/mkdocs-material/ and making small changes to show the Piraeus brand. 

Some changes to the docs were made:
* The markdown parser enforces a "tab length" of 4 for nested lists.
* Various "front pages" were adapted to have a nicer layout, at the cost of some readability in plain markdown.
* Various docs fixes that where noticed during testing.
* Changelog and Upgrade docs have been moved to /docs, so they can be part of the same page.

**Comment by WanzenBug** [comment]
Still a draft: need to change some things in the CI build

**Comment by WanzenBug** [comment]
Now CI works. We automatically update the "latest" v2 mapping on every push to the v2 branch, and automatically update the "stable" mapping on every release.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,981 @@
...
+* Rename CRDs from Piraeus* to Linstor*
+* Make priority classes configurable
+* Fix LINSTOR Controller/Satellite arguments
+* Helm: Make etcd persistent by default
+* Helm: Fix deployment of permissions objects into a non-default namespace
+* Helm: Set default etcd size to 1Gi
+* Helm: Update dependent image versions
+* Docker: Change base image to Debian Buster
+
+## [v0.1.3] - 2020-02-24
```
Comment (JoelColledge) [inline docs/CHANGELOG.md]: Noticed while randomly scrolling around: https://piraeus.io/docs/stable/CHANGELOG/#v013-2020-02-24

3 links are missing.

File: mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    - Networking:
+      - Deploy Piraeus Datastore behind an HTTP Proxy: how-to/http-proxy.md
+      - Deploy a NetworkPolicy for Piraeus Datastore: how-to/network-policy.md
+      - Use the Host Network for DRBD Replication: how-to/drbd-host-networking.md
+    - Maintenance Tasks:
+      - Monitor Piraeus Datastore with Prometheus Operator: how-to/monitoring.md
+      - Keep Persistent Volume Affinity Updated with LINSTOR Affinity Controller: how-to/linstor-affinity-controller.md
+      - Restore a LINSTOR Database Backup: how-to/restore-linstor-db.md
+  - Upgrades:
+    - Upgrades: how-to/upgrade/index.md
```
Comment (JoelColledge) [inline mkdocs.yml]: When I click on "Upgrade" in the navigation I reach the "Upgrading Piraeus Operator from Version 1 to Version 2" page, which doesn't seem right.

File: docs/reference/linstorcluster.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ created by appending an image identifier and tag.
 
 If empty (the default), Piraeus will use `quay.io/piraeusdatastore`.
 
-The current list of default images is available [here](../../config/manager/0_piraeus_datastore_images.yaml).
+The current list of default images is available [here](https://github.com/piraeus/config/manager/0_piraeus_datastore_images.yaml).
```
Comment (JoelColledge) [inline docs/reference/linstorcluster.md]: Page not found.

File: mkdocs.yml [hunk 4]
```diff
@@ -0,0 +1,150 @@
+site_name: Piraeus Datastore
+site_url: https://piraeus.io/docs/
+site_author: The Piraeus Maintainers
+site_description: >-
+  An easy-using cloud native datastore for Kubernetes. 100% open source, run anywhere.
```
Comment (JoelColledge) [inline mkdocs.yml]:
```suggestion
  An easy-to-use cloud native datastore for Kubernetes. 100% open source, run anywhere.
```

File: mkdocs.yml (hunk 2 above)
Comment (JoelColledge) [inline mkdocs.yml]: Why are the upgrade sources in `how-to` even though Upgrade is it's own section?

**Review by JoelColledge — COMMENTED** [review]
It's so beautiful :sunglasses: 

Changes look good. Just some minor nits.

File: docs/CHANGELOG.md (hunk 1 above)
Comment (WanzenBug) [reply docs/CHANGELOG.md]: I noticed that too. I do not see these tags in the repo, so I don't really know what to link :shrug:

File: mkdocs.yml (hunk 2 above)
Comment (WanzenBug) [reply mkdocs.yml]: I guess it was originally to align it with the "4 kinds of documentation" philosophy. I'll move it to it's own section.

File: docs/CHANGELOG.md (hunk 1 above)
Comment (JoelColledge) [reply docs/CHANGELOG.md]: Just remove the link. No-one cares about the release itself, but the `[]` look ugly.

**Review by JoelColledge — APPROVED** [review]

**Pull request merged**